feat: Clarify how coding standard configurations are merged with repository configurations DOCS-508 (#1620)
codacy/docs: docs/organizations/using-a-coding-standard.md

The organization coding standard helps you ensure that Codacy analyzes multiple repositories with the same tool and code pattern settings. For example, you can use the organization coding standard to ensure that all your repositories and teams are following the same security rules or coding conventions.

> When you apply a coding standard, pre-existing repository configurations not defined in the coding standard aren't affected.

Author: ```suggestion
When you apply a coding standard, pre-existing repository configurations not defined in the coding standard aren't affected.
```
```suggestion
When you apply a coding standard, pre-existing repository configurations that are not defined in the coding standard stay the same.
```

> If a coding standard doesn't include a tool, any pre-existing repository settings for that tool aren't modified by the coding standard.

Reviewer: The sentence looks very clear to me. However, it seems like a small detail lost inside this concept piece.
Would it make sense to move this sentence inside the "Important box" below? It would fit as the first bullet of a two-bullet list?
WDYT?

Author: Good point. I agree 👍

> If a coding standard does not include a tool, any pre-existing repository settings for that tool are not modified by the coding standard.

Author: ```suggestion
If a coding standard doesn't include a tool, any pre-existing repository settings for that tool aren't modified by the coding standard.
```

> When you apply a coding standard, pre-existing repository configurations for tools and patterns not included in the coding standard stay the same.

Author: ```suggestion
If a coding standard does not include configurations for a tool, any pre-existing repository configurations for that tool are not modified when you apply the coding standard.
```

> If a coding standard does not include a tool, any pre-existing repository settings for that tool and for the related code patterns are not modified by the coding standard.

Author: ```suggestion
If a coding standard does not include a tool, any pre-existing repository settings for that tool are not modified by the coding standard.
```
